docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+   }
+   ```
+
+3. Aggregate the documentation (see below) from multiple modules into a single output. Add dependencies in the root `build.gradle.kts` file
+   or in the `build.gradle.kts` file from the aggregating subproject directory, if there's any:
+
+   ```kotlin
+   dependencies {
+       dokka(project(":parentProject:childProjectA"))
+       dokka(project(":parentProject:childProjectB"))
+   }
+   ```
+
+### Update documentation aggregation in multi-module projects
+
+Dokka can aggregate the documentation from multiple modules (subprojects) into a single output or publication.
+
+As explained in [Multi-module projects with convention plugins](#multi-module-projects-with-convention-plugins),
+you must apply the Dokka plugin to all documentable subprojects before aggregating the documentation.
+
+Aggregation in DGP v2 is more straightforward. Instead of tasks, now it uses the `dependencies{}` block in any `build.gradle.kts` file.
+
+Previous aggregation:
+
+```kotlin
+tasks.dokkaHtmlMultiModule {
+  // ...
+}
+```
+
+New aggregation:
+
+```kotlin
+dependencies {
+    dokka(project(":some-subproject:"))
+    dokka(project(":another-subproject:"))
+}
+```
+
+### Generate documentation with the updated task
+
+DGP v2 has changed the naming of the Gradle tasks to generate the API documentation.
+
+Previous task:
+
+```kotlin
+./gradlew dokkaHtml
+    
+// OR
+
+./gradlew dokkaHtmlMultiModule
+```
+
+New task:
+
+```kotlin
+./gradlew :dokkaGenerate
+```
+
+In the new version, the `dokkaGenerate` task name works for both single and multi-module projects.
+
+> You can use other tasks to generate output in Javadoc or both HTML and Javadoc. For more information, see
+> [Select documentation output format](#select-documentation-output-format).
+>
+{type="tip"}
+
+After running the `dokkaGenerate` task, the generated API documentation is available in the `build/dokka/` directory.
+
+### Select documentation output format
+
+> The Javadoc output format is in [Alpha](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained).
+> You may find bugs and experience migration issues when using it. Successful integration with tools that accept Javadoc
+> as input is not guaranteed. Use it at your own risk.
+>
+{type="note"}
+
+With DGP v2, you can choose to generate the API documentation in HTML, Javadoc, or both formats at the same time:
+
+1. Place the corresponding plugin `id` in the `plugins{}` block of your project's `build.gradle.kts` file.
+2. Run the corresponding Gradle task.
+
+Here's a list of the plugin `id` and Gradle task that correspond to each format:
+
+|             | **HTML**                       | **Javadoc**                         | **Both**                          |
+|-------------|--------------------------------|-------------------------------------|-----------------------------------|
+| Plugin `id` | `id("org.jetbrains.dokka")`    | `id("org.jetbrains.dokka-javadoc")` | Use both HTML and Javadoc plugins |
```

Comment:
@adam-enko Could you please confirm:  Can I leave `./gradlew :dokkaGenerate` instead of `./gradlew :dokkaGenerateHtml` when the plugin is `id("org.jetbrains.dokka")` and the desired output is HTML?

Reply:
`dokkaGenerate` is a 'lifecycle' task that will generate all present formats.

So if a project has both `org.jetbrains.dokka` and `org.jetbrains.dokka-javadoc` plugins then running `./gradlew dokkaGenerate` will generate both HTML and Javadoc outputs.

If a project only has `org.jetbrains.dokka`, then `./gradlew dokkaGenerate` will only generate HTML. The `dokkaGenerateJavadoc` task will not be present.

I think the current docs are fine, and I'm happy with any other modifications. My weak suggestion is to keep the documentation for `dokkaGenerateHtml` and `dokkaGenerateJavadoc`. It helps demonstrate that DGPv2 has been split, and users can see the cause/effect of the plugins by checking the available tasks, or running them to see what happens.

Reply:
I added a note explaining the behavior of `dokkaGenerate` and explaining the  option of generating only one format with the two plugins applied. I think this options was not explained. Please take a look.